blog/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+
+from .views import index, PostDetailView, comment_create, PostCreateView
+
+urlpatterns = [
+    path("", index, name="index"),
+    path("post/<int:pk>/", PostDetailView.as_view(), name="post-detail"),
+    path(
+        "post/<int:pk>/comment/",
+        comment_create,
+        name="comment-create"
+    ),
+    path("post/create", PostCreateView.as_view(), name="post-create"),

```

Comment:
check the correct naming of urls in the requirements + add / at the end of each url